javascriptv3/example_code/medical-imaging/tests/copy-image-set.unit.test.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {describe, it, expect, vi} from "vitest";
+
+const send = vi.fn();
+
+vi.doMock("@aws-sdk/client-medical-imaging", async () => {
+    const actual = await vi.importActual("@aws-sdk/client-medical-imaging");
+    return {
+        ...actual,
+        MedicalImagingClient: class {
+            send = send;
+        },
+    };
+});
+
+const {copyImageSet} = await import("../actions/copy-image-set.js");
+
+describe("copy-image-set", () => {
+    const response =
```

Comment:
Because you're not doing anything with `response` in the function under test, you don't need an accurate object here. You could trim it down to something like
```suggestion
    const response = { a: 'a' }
```

Reply:
I realized this when I had already coded some tests. I suppose there never will be any response checking sometime in the future which would break the tests.